File: espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/model/StartActivityParams.java
package io.appium.espressoserver.lib.model;

import javax.annotation.Nullable;

@SuppressWarnings("unused")
public class StartActivityParams extends AppiumParams {
    private String appActivity;
    private String appWaitActivity;

    @Nullable
    public String getAppActivity() {
        return appActivity;
    }

    @Nullable
    public String getAppWaitActivity() {
        return appWaitActivity;
    }
}

File: espresso-server/library/src/main/java/io/appium/espressoserver/lib/model/StartActivityParams.kt
package io.appium.espressoserver.lib.model

data class StartActivityParams(
    val appPackage: String? = null,
    val appActivity: String? = null,
    val locale: Map<String, Any?>? = null,
    val optionalIntentArguments: Map<String, Any?>? = null,
    val optionalActivityArguments: Map<String, Any?>? = null
) : AppiumParams()
